spec/e2e/id_property_spec.rb

```diff
@@ -64,7 +64,7 @@
           Neo4j::Config[:id_property] = :the_id
           Neo4j::Config[:id_property_type] = :auto
           Neo4j::Config[:id_property_type_value] = :uuid
-          stub_const('Clazz', Class.new do
+          stub_const('Clazz', UniqueClass.create do
```

Comment:
Actually `stub_active_node_class` is just a shortcut for what was there before. Since constraints are not cleaned up before each test, some other tests are bleeding into this specs and make it fail if the class is not unique.